### ISSUE TITLE
Support per-tab show/hide binding

### ISSUE THREAD

**Issue by fredemmott** [OpenKneeboard/OpenKneeboard#538]
Not going to do this until I've pretty much rewritten the binding system.

**Comment by fredemmott** [comment]
I was going to add a remote control for this, but it turned out to be more complicated:
- hiding or toggling may change the 'active' view, both for the game and for the app window
- it will be possible for there to be *no* active view, which is not currently something that OpenKneeboard allows or supports

**Comment by svengiesen** [comment]
Wouldn't be no active view the same as `OpenKneeboard-RemoteControl-HIDE.exe: hide all views` that could be detected conditionally?

**Comment by fredemmott** [comment]
No, those toggle all interprocess sharing off/on and don't otherwise affect internal state.

**Comment by fredemmott** [comment]
Some indirect progress on handling 'no current view' with the various 'crash when siwtching profiles' fixes + 993d8a27ad6ee9e7af7542d7a1e7516cf604ddf2 and de0928816efe4a4260fd6c0e3da23124a2e99560, but this still isn't something I'm directly working on

**Comment by marksdoran** [comment]
Alternative approach that might be easier??  I would like to have a way to blank any of the <N> kneeboards that I have set up as independent in VR.  I use the next/prev bindings to change which tab content appears on the various kneeboards.  If it were possible to create a tab that shows blank space, instead of the big white box and NO PAGES, using next/prev to cycle the view to the blank one would be the moral equivalent of per-view binding without creating something new.

**Comment by fredemmott** [comment]
Add an empty or fully transparent png.
